<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chakanClass</class>
 <widget class="QMainWindow" name="chakanClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>查看题目</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>请选择需要查看的题目</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>380</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>380</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>题目：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>280</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>答案：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_A">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>150</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>A.147</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_B">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>180</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>B.157</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_C">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>210</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>C.167</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_D">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>240</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>D.177</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_timu">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>441</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>123+45=</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_answer">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>310</y>
      <width>441</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>请输入题库所在文件夹</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>391</width>
      <height>231</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>选择文件夹..</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
